src/crypter.js

```diff
@@ -32,6 +34,7 @@ module.exports.Crypter = (secret: Buffer, {
     throw new TypeError();
   }
 
+  // The point is generate a potentially longer passphrase?
```

Comment:
The passphrase needs to have a certain length. This is the easiest and safest way to do that.

Reply:
any consideration to using a key derivation function that's resistant to attack? Something like `Crypto.pbkdf2`

Reply:
There is really no point here. The secret has to be known. This is just a different representation of it, not a way to check if it is valid.

Reply:
added a comment